src/App.tsx

```diff
@@ -1,61 +1,122 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
```

Comment:
The `usersFromServer` import is used directly in the `App` component to prepare data. This violates the requirement to prepare data in one place and pass it to child components. Consider refactoring to ensure data is prepared and passed correctly.